### PR TITLE
feat: translated to Spanish Getting started section

### PR DESCRIPTION
**Pull request by mmorejon**

### CODE REVIEW

**Comment by mmorejon** [comment]
Vamos por más! Estoy ahora en la sección Concepts!

File: content/es/docs/getting-started/demos-talks-posts/articles.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
```
Comment (vfarcic) [inline content/es/docs/getting-started/demos-talks-posts/articles.md]: I don't think it makes sense to translate this page, unless someone wants to commit to keeping it translated continuously.

File: content/es/docs/getting-started/demos-talks-posts/talks.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
+---
```
Comment (vfarcic) [inline content/es/docs/getting-started/demos-talks-posts/talks.md]: This is a live page that is being updated continuously. Can someone commit to keeping it always up to date? If not, it's better to leave it in english.

File: content/es/docs/getting-started/demos-talks-posts/articles.md (hunk 1 above)
Comment (sharepointoscar) [reply content/es/docs/getting-started/demos-talks-posts/articles.md]: Yes, agreed.  @mmorejon maybe these can stay the way they are?  Also less work maintaining the translation if we don't bother translating these...

File: content/es/docs/getting-started/demos-talks-posts/talks.md (hunk 2 above)
Comment (sharepointoscar) [reply content/es/docs/getting-started/demos-talks-posts/talks.md]: I recommend not translating this particular page.

Comment (mmorejon) [reply content/es/docs/getting-started/demos-talks-posts/talks.md]: Sounds good for me. One question, whats happens with the `es` folder structure?
- should I remove the file completely? Will not appear in the navigation bar.
- should I keep some reference to the generated page in the `en` folder?

File: content/es/docs/getting-started/demos-talks-posts/articles.md (hunk 1 above)
Comment (mmorejon) [reply content/es/docs/getting-started/demos-talks-posts/articles.md]: Good. This case is similar to the question asked in the following conversation.

File: content/es/docs/getting-started/demos-talks-posts/talks.md (hunk 2 above)
Comment (mmorejon) [reply content/es/docs/getting-started/demos-talks-posts/talks.md]: @vfarcic @sharepointoscar ?

**Pull request merged**